.gitattributes

```diff
@@ -1 +1,3 @@
 *.sh text eol=lf
+./public/static/dist/* binary
```

Comment:
I thought I can forbid git treat these files as text and show diffs for them. Because I don't think we interested in these changes at all. If we will add some minfication, these file will look closer to binary, then to text.